integrations/fastembed/src/haystack_integrations/components/embedders/fastembed/fastembed_document_SPLADE_embedder.py

```diff
@@ -0,0 +1,171 @@
+from typing import Any, Dict, List, Optional
+
+from haystack import Document, component, default_to_dict
+
+from .embedding_backend.fastembed_backend import _FastembedSparseEmbeddingBackendFactory
+
+
+@component
+class FastembedDocumentSPLADEEmbedder:
```

Comment:
`FastembedDocumentSPLADEEmbedder` should be `FastembedDocumentSparseEmbedder`?
Since it won't be limited to SPLADE models only But will have other sparse models as well.

Reply:
Complete Renaming done :)